test/travis_commands.jl

```diff
@@ -1,8 +1,8 @@
-# Pkg.clone("ImageMagick")
-# Pkg.build("ImageMagick")
+Pkg.clone("ImageMagick")
```

Comment:
it'll be more representative of what users run with if you do Pkg.add and test against releases, instead of clone and master